atd-vzd/migrations/default/1715094801442_add_import_log_table/up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE cris_import_log (
+    id SERIAL PRIMARY KEY,
+    object_path TEXT NOT NULL,
+    object_name TEXT NOT NULL,
+    created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    completed_at TIMESTAMPTZ DEFAULT NULL,
+    records_processed JSONB DEFAULT '{}',
+    outcome_status TEXT NULL
```

Comment:
Again, repeating something from way above -- these two columns are /not/ getting populated in the code at the time of writing, and I'd be very happy to remove them as cruft. We can create spin off issues to enhance this code to populate them, but really, we should focus that effort on the CRIS importer in the new data model, perhaps?

Reply:
You bet, great idea. Here are some comments for each column in this table. Thanks! https://github.com/cityofaustin/atd-vz-data/pull/1445/commits/e2ca36ef9fd1b427e422502bfb29c3c720d81bc9